src/Symfony/Component/Ldap/Entry.php

```diff
@@ -22,9 +22,24 @@ class Entry
     public function __construct(string $dn, array $attributes = array())
     {
         $this->dn = $dn;
+        $attributes = $this->resetValuesIndex($attributes);
         $this->attributes = $attributes;
     }
 
+    /**
+     * Resets the index for array values.
+     *
+     * @param array $attributes
+     *
+     * @return array
+     */
+    private function resetValuesIndex(array $attributes = array())
+    {
+        return array_map(function ($o) {
```

Comment:
Rename this to `$v` to indicate that you are handling values. The most commonly used convention is `$k` for keys, and `$v` for values.

Reply:
done.